apidoc/Titanium/Media/Media.yml

```diff
@@ -1749,6 +1767,13 @@ properties:
     default: true
     since: "3.4.0"
     platforms: [ipad]
+  - name: focusMode
+    summary: Constant to indicate focus mode of camera.
+    constants: Titanium.Media.FOCUS_MODE_*
+    type: Number
+    default: Depends on camera
```

Comment:
Can we write something else here? It doesn't really help the developer. So either more infos on how it depends on the camera or leave it out and add a link to the Apple docs in the `description` value.